src/main/kotlin/blackjack/model/BlackjackParticipant.kt

```diff
@@ -1,11 +1,21 @@
 package blackjack.model
 
-sealed class BlackjackParticipant(val name: PlayerName, cardDeck: CardDeck) {
+sealed class BlackjackParticipant(cardDeck: CardDeck) {
     val handDeck: HandDeck = HandDeck()
 
     val deckScore: Int
         get() = handDeck.score
 
+    val deckCount: Int
+        get() = handDeck.count
+
+    fun isWinByScoreGap(participant: BlackjackParticipant): Boolean {
+        return (HandDeck.LIMIT_SCORE - deckScore) < (HandDeck.LIMIT_SCORE - participant.deckScore)
+    }
+
+    val isSameLimitScore: Boolean
+        get() = (HandDeck.LIMIT_SCORE == deckScore)
```

Comment:
`deckScore`, `HandDeck` 모두 `handDeck`이 잘 알고있으니, 다음과 같은 코드가 좀 더 응집도를 높히겠네요~ 비슷한 유형도 수정해볼까요?

```suggestion
    val isSameLimitScore: Boolean
        get() = handDeck.isLimitScore()
```

Reply:
넵 수정하도록 하겠습니다!